docs/_data/icons.yaml

```diff
@@ -456,6 +456,7 @@ icons:
     - sort-up-12
     - split-view-24
     - split-view-filled-24
+    - star-dynamic
```

Comment:
I don't think this icon needs to be included. It's the icon we use to create ratings dynamically. This wouldn't make sense as a standalone icon.